admin/javascript/src/components/grid-field/row.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SilverStripeComponent from 'silverstripe-component';
+
+class GridFieldRowComponent extends SilverStripeComponent {
+
+  render() {
+    return (
+      <li className="grid-field-row-component [ list-group-item ]">{this.props.children}</li>
```

Comment:
@scott1702 can you explain please? I had the same concerns as Hamish. From memory, you wanted to ensure that tables can be rearranged responsively (e.g. through http://v4-alpha.getbootstrap.com/content/tables/#responsive-tables). I also think `<table>` is more semantic in this case, rather than `<ul>/<li>` with random `<div>` scattered in them as "cells"